pyrobot/__init__.py

```diff
@@ -8,7 +8,7 @@
 if bool(os.environ.get("ENV", False)):
     from pyrobot.sample_config import Config
 else:
-    from pyrobot.config import Development as Config
+    from pyrobot.sample_config import Development as Config
```

Comment:
what?

Reply:
no need rename sample_config to config..

Reply:
?